rs/sns/governance/src/request_impls.rs

```diff
@@ -1,82 +1,79 @@
 use ic_nervous_system_clients::Request;
 
-use crate::pb::v1::{
-    ClaimSwapNeuronsRequest, ClaimSwapNeuronsResponse, FailStuckUpgradeInProgressRequest,
-    FailStuckUpgradeInProgressResponse, GetMaturityModulationRequest,
-    GetMaturityModulationResponse, GetMetadataRequest, GetMetadataResponse, GetMode,
-    GetModeResponse, GetNeuronResponse, GetProposalResponse, GetRunningSnsVersionResponse,
-    GetSnsInitializationParametersRequest, GetSnsInitializationParametersResponse,
-    ListNeuronsResponse, ListProposalsResponse, ManageNeuronResponse,
-};
-
-impl Request for ClaimSwapNeuronsRequest {
-    type Response = ClaimSwapNeuronsResponse;
+impl Request for crate::pb::v1::ClaimSwapNeuronsRequest {
```

Comment:
Are these used anywhere?  I'm not comfortable with these being defined in the governance canister crate, as we want to separate out the api from the internals to avoid dependencies of these types needing to pull in the whole governance crate.  If we're planning to make this a reusable pattern, we really need to do the separation of the API types into their own crate.

Reply:
Discussed over zoom, we'll move these when we make the SNS Gov API crate